Add a note about usecases for the clone and backup tools for 3.1 (#2191)
theforeman/foreman-documentation: guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc

= Cloning {ProjectServer}

You can clone the {ProjectServer} to create  instances for trying out upgrades and the migration of instances to a different machine or operating system.
This is an optional step to provide more flexibility during the upgrade.

> [NOTE]

Reviewer: ```suggestion

[NOTE]
```
Same applies to above.

= Cloning {ProjectServer}

> You can clone the {ProjectServer} to create  instances for trying out upgrades and the migration of instances to a different machine or operating system.

Reviewer 2: ```suggestion
You can clone the {ProjectServer} to create instances for trying out upgrades and the migration of instances to a different machine or operating system.
```

= Cloning {ProjectServer}

You can clone the {ProjectServer} to create  instances for trying out upgrades and the migration of instances to a different machine or operating system.
This is an optional step to provide more flexibility during the upgrade.

> Note: The created instances are not supposed to run in parallel in a production environment, please decommission any testing instances after completing the tests or any old instances after completing the migration.

Reviewer 2: ```suggestion
[NOTE]
----
The created instances are not supposed to run in parallel in a production environment.
Please decommission any testing instances after completing the tests or any old instances after completing the migration.
----
```

Reviewer 3: IMHO it must be `====` for admonitions: https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/#admonition-syntaxhttps://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/#admonition-syntax

> Note: The created instances are not supposed to run in parallel in a production environment, please decommission any testing instances after completing the tests or any old instances after completing the migration.

Reviewer: ```suggestion
Note: The created instances are not supposed to run in parallel in a production environment. You must decommission any testing instances after completing the tests or any old instances after completing the migration.
```
Agree to Adam's view + We do not use polite terms.

Reviewer 3: Please stick to one sentence per line.
